tools/picard/picard_CollectSequencingArtifactsMetrics.xml

```diff
@@ -0,0 +1,53 @@
+<tool id='picard_artifact_metrics' name='Picard Collect Sequencing Artifact Metrics'>
+  <description>writes 5 tables containing Phred-scaled error rates substitution details and summaries</description>
+  <!--<requirements>
+    <requirement type="package">picard</requirement>
+  </requirements>-->
+  <command>
+    <![CDATA[
+    
+    ##set up input files
+
+    #set $reference_fasta_filename = "/Users/aerijman/Documents/projects/galaxy/tool-data/hg38/seq/hg38.fa"
+    
+    picard -Xmx4048m
+    CollectSequencingArtifactMetrics
+    I='${sam_bam}'
+    O='${output_name}'
+    R='/Users/aerijman/Documents/projects/galaxy/tool-data/hg38/seq/hg38.fa' 
+    CONTEXT_SIZE=${context_size}
+    CONTEXTS_TO_PRINT="${context_print}"
+    INCLUDE_DUPLICATES="${duplicates}"
+    MINIMUM_QUALITY_SCORE="${min_quality_score}";
+
+    ]]>
+  </command>
+
+  <stdio>
+    <exit_code range="1:"  level="fatal" />
+  </stdio>
+  
+  <inputs>
+    <!--<param name='output_name' type='text' label='prefix to name output files'>-->
+    <param name='sam_bam' type='data' label='Sam/Bam file'/>
+    <param name='reference_fasta_filename' type='data' label='Reference genome' help='last reference genome used is default'/>
```

Comment:
https://github.com/galaxyproject/tools-iuc/blob/64305ae905dc79e44b478203000add73598bcca1/tools/picard/picard_ValidateSamFile.xml#L57

Reply:
Awesome!
After you review the new version I can delete the newly commented line for the genome if you approve.